.github/workflows/integration-tests-dev.yml

```diff
@@ -5,6 +5,9 @@ env:
   TURBO_TEAM: ${{ vars.TURBO_TEAM }}
   POSTGRES_URL: ${{ secrets.POSTGRES_URL }}
   POSTGRES_URL_NON_POOLING: ${{ secrets.POSTGRES_URL_NON_POOLING }}
+  BLOB_READ_WRITE_TOKEN: ${{ secrets.BLOB_READ_WRITE_TOKEN }}
+  KV_REST_API_URL: ${{ secrets.KV_REST_API_URL }}
```

Comment:
Why do we need these?

Reply:
the `integration-tests-dev.yml` is running on a local nextjs instance, I guess it's because we want to test that the packages work on a local dev

Reply:
I meant why did you had to add "KV_REST_API_URL" while this PR is for blob? Because it was missing and failing probably?

Reply:
not failing as we don't have tests yet, I just prepared the store and envs to add them

Reply:
Got it 👍